cli/src/main/scala/filodb.cli/CsvImportExport.scala

```diff
@@ -57,51 +54,15 @@ trait CsvImportExport {
   def ingestCSV(dataset: String, version: Int, csvPath: String, delimiter: Char) {
     val fileReader = new java.io.FileReader(csvPath)
 
-    val reader = new CSVReader(fileReader, delimiter)
-    val columns = reader.readNext.toSeq
-    println(s"Ingesting CSV at $csvPath with columns $columns...")
-
-    val ingestCmd = CoordinatorActor.SetupIngestion(dataset, columns, version: Int)
-    actorAsk(coordinatorActor, ingestCmd, 10 seconds) {
-      case CoordinatorActor.IngestionReady =>
-      case CoordinatorActor.UnknownDataset =>
-        println(s"Dataset $dataset is not known, you need to --create it first!")
-        exitCode = 2
-        return
-      case CoordinatorActor.UndefinedColumns(undefCols) =>
-        println(s"Some columns $undefCols are not defined, please define them with --create first!")
-        exitCode = 2
-        return
-      case CoordinatorActor.BadSchema(msg) =>
-        println(s"BadSchema - $msg")
+    val csvActor = system.actorOf(CsvSourceActor.props(fileReader, dataset, version, coordinatorActor))
```

Comment:
Do we want all instances of csvActor to not have a supervisor? For failures etc.
I mean, by doing
 
     system.actorOf(...)

versus this in a supervisor actor

     context.actorOf(..) 

just a thought.